apps/sms/views/inbox/js/inbox.js

```diff
@@ -39,6 +39,11 @@ var InboxView = {
   // Set to |true| when in edit mode
   inEditMode: false,
 
+  /**
+   * Indicates whether user should be notified about newly saved draft.
+   */
+  notifyAboutSavedDraft: false,
```

Comment:
It's already hard to recover that in memory, but iirc, here is the reason [1] :)

> The problem is that I've moved draft manipulation code to beforeLeave so I don't have access to that info (saved or not) when I navigate to Inbox. 


[1] https://github.com/mozilla-b2g/gaia/pull/30898#discussion_r34276785